stdlib/private/StdlibCollectionUnittest/CheckSequenceType.swift

```diff
@@ -136,6 +136,50 @@ public struct FindTest {
   }
 }
 
+public struct CollectionTypeTest {
```

Comment:
Could you call this `CollectionBinaryOperationTest`?  I think it better reflects what it is.